src/gui/src/viewer/zoom-window.cpp

```diff
@@ -220,6 +220,12 @@ void ZoomWindow::go()
 	m_detailsWindow = new DetailsWindow(m_profile, this);
 	colore();
 
+	QShortcut *accept = new QShortcut(getKeySequence(m_settings, "keyAcceptDialogue", Qt::CTRL + Qt::Key_Y), m_detailsWindow);
+		//connect(accept, &QShortcut::activated, m_detailsWindow, &QDialog::done(0));
```

Comment:
This isn't really important but I was curious how QDialog::done(int) is supposed to be used. I couldn't get it to work.

Reply:
You're not supposed to use QDialog::done(int) but QDialog::accept() and QDialog::reject().
With accept being equivalent to the "Ok" button, and reject being equivalent to "Cancel".

Reply:
And actually currently  the button just uses close() so you can just use that.